staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/aggregator.go

```diff
@@ -42,6 +46,7 @@ type SpecAggregator interface {
 	AddUpdateAPIService(handler http.Handler, apiService *v1.APIService) error
 	UpdateAPIServiceSpec(apiServiceName string, spec *spec.Swagger, etag string) error
 	RemoveAPIServiceSpec(apiServiceName string) error
+	UpdateOpenAPIV3(downloader Downloader, apiService string, handler http.Handler) error
```

Comment:
This looks very odd. All other funcs are SomethingAPIServiceSpec. Why is this different? document what the func does.